src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -206,9 +213,40 @@ private static String getUserIdUrl(String encodedUsername) {
       new UrlBuilder().getUrlForWiki("community"), encodedUsername, DateTime.now().getMillis());
   }
 
+  private static X509TrustManager getPassiveTrustManager() {
+    return new X509TrustManager() {
+
+      public X509Certificate[] getAcceptedIssuers() {
+        return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty array instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1168)